.github/workflows/unit_test.yml

```diff
@@ -0,0 +1,26 @@
+name: Unit test
+
+on:  
+  pull_request:
+    branches:
+    - master
+    paths:
+    - 'data_classes.py'
```

Comment:
Please add data_classes_test.py as well to make sure changes in test code are correct.